action.yml

```diff
@@ -3,6 +3,7 @@ branding:
   icon: shopping-bag
   colour: green
 description: 'Sends a request to GitHub in order to track deployment'
+deprecationMessage: 'This action is now deprecated. See https://github.com/Shopify/github-deployment-action?tab=readme-ov-file#deprecation-notice for more information.'
```

Comment:
Does this work? If I remember correctly, for `oxygenctl-action` we had to add this to one of the inputs in order for it to show the notice on usage ([here](https://github.com/Shopify/oxygenctl-action/blob/7fe2cc0dde5aa3b971e161b12a384fea272487bc/action.yml#L21)).